src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -829,44 +825,6 @@ using stan::model::index_multi;
 using stan::model::index_omni;
 using stan::model::nil_index_list;
 using namespace stan::math;
-using stan::math::pow; |}
```

Comment:
- using std::istream; - not used
- using std::string and using std::stringstream; - only one use that now has std::
- using std::pow; and using stan::math::pow; - these were temporary workaround that should be resolved now.
- using stan::math::lgamma; - same here